common/containers/Tabs/GenerateWallet/index.tsx

```diff
@@ -38,38 +39,43 @@ class GenerateWallet extends Component<Props, {}> {
 
     const AnyEnterPassword = EnterPassword as new () => any;
 
-    switch (activeStep) {
-      case 'password':
-        content = (
-          <AnyEnterPassword
-            walletPasswordForm={this.props.walletPasswordForm}
-            generateNewWallet={this.props.generateNewWallet}
-          />
-        );
-        break;
-
-      case 'download':
-        if (wallet) {
+    if (window.crypto) {
```

Comment:
Has this been tested with browserstack? 

I would be inclined to believe calling window.crypto would throw an exception instead of returning falsy. In this case we would want try/catch instead.